riff-raff.yaml

```diff
@@ -46,3 +47,16 @@ deployments:
       bucket: aws-frontend-static
       cacheControl: public, max-age=315360000
       prefixStack: false
+  update-ami:
+    type: ami-cloudformation-parameter
+    parameters:
+      cloudFormationStackByTags: false
+      prependStackToCloudFormationStackName: false
+      cloudFormationStackName: frontend
+      amiParametersToTags:
+        AMI:
+          Recipe: frontend-base
+          AmigoStage: PROD
+        RouterAMI:
+          Recipe: frontend-router
+          AmigoStage: PROD
```

Comment:
It would make more sense to update the RouterAMI when deploying the router (ie: in platform repo) rather than here @DiegoVazquezNanini @gtrufitt